### ISSUE TITLE
Please add a brief but descriptive title

### ISSUE THREAD

**Issue by iSeekerTW** [cabaletta/baritone#2877]
## Some information
Operating system: macOS Big Sur 11.3.1 (M1 chip)
Java version: Java 8 Update 291
Minecraft version: 1.16.5
Baritone version: v1.6.3
Other mods (if used):  Sodium, Lithium, Phosphor, BleachHack, Litematica, WorldEdit, Tweakeroo, MiniHUD, malilib

## Exception, error or logs
[latest.log](https://github.com/cabaletta/baritone/files/6835492/latest.log)

## How to reproduce
1. build a schematic 
2. Turn of BuildInLayers (bot builds it without an issue)
3. Turn on BuildInLayers
4. Bot started to stand there and pathing forever. 

please watch this video:

https://user-images.githubusercontent.com/56954729/126052562-e1b3b55a-ec0d-48c6-b011-4c73d36fd582.mp4

## Modified settings
All the modified settings are shown in the video
But here's a list anyway:
AllowInventory True
BlockPlacementPenalty 20.0
BuildIgnoreBlocks air
BuildIgnoreExisting true
RightClickSpeed 2

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by iSeekerTW** [comment]
![37CF31D7-B169-4C34-8B77-03AFEAEBFD53](https://user-images.githubusercontent.com/56954729/126052892-2d1a6082-295b-480f-9077-1d1ba2254522.png)
In case you are wondering, this is the build I’m trying to create. I have an entire dirt platform below the island, and I was building it with “build in layer” fine for several hours until it suddenly failed to pathfind. 

Since it’s built in layers, it’s a pretty flat surface, and I have no idea why it would fail to path find in such scenario

**Comment by iSeekerTW** [comment]
<img width="1440" alt="截屏2021-07-18 上午10 13 52" src="https://user-images.githubusercontent.com/56954729/126053604-0ac653ca-f1e7-47f1-8a91-690bce783606.png">
This is my progress so far

**Comment by iSeekerTW** [comment]
Theres a new issue: it somehow said I don't have "spruce leaves", and said im missing materials. But I clearly have 64 spruce leaves in my inventory. How is that the case

**Comment by scorbett123** [comment]
Make sure that the schematic was made  on the same mc version as you are on now.

**Comment by iSeekerTW** [comment]
Yes it is. I made the schematic using world edit on my 1.16.5 single player world

**Comment by scorbett123** [comment]
Can you try setting the layerOrder to false?

**Comment by iSeekerTW** [comment]
It is set to false already
